src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/MethodConfigurationComponent.scala

```diff
@@ -74,8 +77,155 @@ trait MethodConfigurationComponent {
     def configKeyIdx = index("IDX_MC_PREREQ", (methodConfigId, key), unique = true)
   }
 
-  protected val methodConfigurationQuery = TableQuery[MethodConfigurationTable]
   protected val methodConfigurationInputQuery = TableQuery[MethodConfigurationInputTable]
   protected val methodConfigurationOutputQuery = TableQuery[MethodConfigurationOutputTable]
   protected val methodConfigurationPrereqQuery = TableQuery[MethodConfigurationPrereqTable]
+
+  object methodConfigurationQuery extends TableQuery(new MethodConfigurationTable(_)) {
+
+    private type MethodConfigurationQueryType = driver.api.Query[MethodConfigurationTable, MethodConfigurationRecord, Seq]
+    private type MethodConfigurationInputQueryType = driver.api.Query[MethodConfigurationInputTable, MethodConfigurationInputRecord, Seq]
+    private type MethodConfigurationOutputQueryType = driver.api.Query[MethodConfigurationOutputTable, MethodConfigurationOutputRecord, Seq]
+    private type MethodConfigurationPrereqQueryType = driver.api.Query[MethodConfigurationPrereqTable, MethodConfigurationPrereqRecord, Seq]
+
+    /*
+      the core methods
+     */
+    def save(workspaceId: UUID, methodConfig: MethodConfiguration): ReadWriteAction[MethodConfiguration] = {
```

Comment:
all of these core DAO methods have a UUID as a param. Will be changed to a SlickWorkspaceContext once I rebase off of the EntityDAO code